daprdocs/content/en/reference/components-reference/supported-configuration-stores/azure-appconfig-configuration-store.md

```diff
@@ -0,0 +1,93 @@
+---
+type: docs
+title: "Azure App Configuration"
+linkTitle: "Azure App Configuration"
+description: Detailed information on the Azure App Configuration configuration store component
+aliases:
+  - "/operations/components/setup-configuration-store/supported-configuration-stores/setup-azure-appconfig/"
+---
+
+## Component format
+
+To set up an Azure App Configuration configuration store, create a component of type `configuration.azure.appconfig`.
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+spec:
+  type: configuration.azure.appconfig
+  version: v1
+  metadata:
+  - name: host # host should be used when 
```

Comment:
based on the code this should be `appConfigHost`

Reply:
good catch!